runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/InfoBoxOverlay.java

```diff
@@ -83,11 +84,10 @@ public Dimension render(Graphics2D graphics)
 		}
 
 		panelComponent.getChildren().clear();
-		panelComponent.setWrapping(config.infoBoxWrap());
+		panelComponent.setPreferredSize(new Dimension(config.infoBoxSize(), config.infoBoxSize()));
```

Comment:
This appears to be causing the infoboxes to wrap at 2 (although it reads like it would make it wrap after 1..?). Regardless it should stay consistent with the current behavior and wrap at 4 by default.